style.css

```diff
@@ -43,14 +43,35 @@ code {
  * Styles
  */
 body {
+	display: grid;
+	grid-template:
+		'curve header header' auto
+		'curve preview library' auto
+		'curve . .' auto / 300px 480px 1fr;
+	gap: 2rem;
 	position: relative;
 	margin: 1em 5% 0;
-	padding-left: 330px;
 	background: white;
 	font-family: 'Hiragino Kaku Gothic Pro', 'Segoe UI', 'Apple Gothic', Tahoma, 'Helvetica Neue', sans-serif;
 	line-height: 1.4;
 }
 
+header {
```

Comment:
Not sure if you'd want them together with their other declarations or this, so let me know what you'd prefer here

Reply:
I'd prefer them together with their other declarations if that's ok